Project/Assets/Prairie/Framework/Script/Twine/TwineNode.cs

```diff
@@ -15,14 +19,43 @@ public class TwineNode : MonoBehaviour {
 	[HideInInspector]
 	public string[] childrenNames;
 	public List<GameObject> parents = new List<GameObject> ();
+	public bool isDecisionNode;
 
 	private bool isMinimized = false;
+	private bool isOptionsGuiOpen = false;
+
+	private int selectedOptionIndex = 0;
+
+	void Start () 
+	{
+		// Upon creation of this node, ensure that it is a decision node if it has
+		//	the decision tag:
+		this.isDecisionNode = this.tags.Contains (PRAIRIE_DECISION_TAG);
```

Comment:
My bad -- I was thinking the `Start()` method gets called on creation, not on play.

Will add this logic to the Twine importer instead.